spread_tests/lib/prepare.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+. $TESTSLIB/utilities.sh
+
+if [ $SPREAD_REBOOT -gt 0 ] ; then
+
+    # after reboot, configure WLAN radio interfaces
+    modprobe mac80211_hwsim radios=2
+    wait_until_interface_is_available wlan0
+    wait_until_interface_is_available wlan1
+
+    # Powercycle both interface to get them back into a sane state before
+    # we install the wifi-ap snap
+    snap install --devmode wireless-tools
```

Comment:
that is needed for wireless-tools have permissions to access RFKILL control device (/dev/rfkill)